README.rst

```diff
@@ -45,6 +45,15 @@ them yourselves using your own Python scripts.
 Module and CLI Guide
 ~~~~~~~~~~~~~~~~~~~~
 
+If you have the project installed in a virtualenv, you can use the following console scripts:
```

Comment:
Might be more accurate to say "if you've installed the module with `pip`", since it doesn't really matter if it's in a virtualenv or in their system site-packages, as long as it was installed.